components/Modal.tsx

```diff
@@ -19,13 +25,19 @@ const Modal: FC<ModalProps> = ({ actionButtons, children, open }) => {
       >
         <div
           role="dialog"
-          aria-describedby={`${id}-modal-desc`}
-          className="mx-4 p-4 bg-white border-2 border-black md:w-2/3 lg:w-2/5"
+          aria-describedby={`${id}-modal-header`}
+          className="bg-white ring-2 ring-gray-modal md:w-2/3 lg:w-2/5 rounded"
```

Comment:
`mx-4` class is important for mobile view... othewise the model is full width... should have x-axis margin

Reply:
good catch on this 😄